invenio_curations/requests/curation.py

```diff
@@ -23,7 +27,7 @@
 class CurationCreateAndSubmitAction(actions.CreateAndSubmitAction):
     """Create and submit a request."""
 
-    def execute(self, identity, uow):
+    def execute(self, identity: Any, uow: Any) -> None:
```

Comment:
i think here it is not necessary to use Any

from flask_principal import Identity
from invenio_db import UnitOfWork

identity: Identity
uow: UnitOfWork